src/cli/onefuzz/api.py

```diff
@@ -153,6 +158,42 @@ def _req_model_list(
 
         return [model.parse_obj(x) for x in response]
 
+    def _req_config_params(
+        self,
+    ) -> None:
+
+        endpoint_params = responses.Config(
```

Comment:
No need for this I think since we are overwriting this variable later?

Reply:
Removed.